### PR TITLE
v2v:add ESXi8.0 conf info in vt

### PR DESCRIPTION
**Pull request by vwu-vera**
Signed-off-by: vwu-vera <vwu@redhat.com>

### CODE REVIEW

**Comment by vwu-vera** [comment]
@xiaodwan please help to merge. Thanks. :)

File: virttest/backends/v2v/cfg/convert_source.cfg [hunk 1]
```diff
@@ -37,4 +37,10 @@ variants:
...
                 esx_hostname = ESX_70_HOSTNAME_V2V_EXAMPLE
-                vddk_libdir_src = MOUNT_SRC_VDDK70_LIB_DIR_V2V_EXAMPLE
\ No newline at end of file
+                vddk_libdir_src = MOUNT_SRC_VDDK70_LIB_DIR_V2V_EXAMPLE
+            - esx_80:
+                datastore = VPX_80_DATASTORE_V2V_EXAMPLE
+                vpx_hostname = VPX_80_HOSTNAME_V2V_EXAMPLE
+                vpx_dc = VPX_80_DC_V2V_EXAMPLE
+                esx_hostname = ESX_80_HOSTNAME_V2V_EXAMPLE
+                vddk_libdir_src = MOUNT_SRC_VDDK70_LIB_DIR_V2V_EXAMPLE
```
Comment (xiaodwan) [inline virttest/backends/v2v/cfg/convert_source.cfg:46]: what vddk version is it? vddk80 or 70?

File: virttest/backends/v2v/cfg/convert_source.cfg [hunk 2]
```diff
@@ -37,4 +37,10 @@ variants:
                 vpx_hostname = VPX_70_HOSTNAME_V2V_EXAMPLE
                 vpx_dc = VPX_70_DC_V2V_EXAMPLE
                 esx_hostname = ESX_70_HOSTNAME_V2V_EXAMPLE
-                vddk_libdir_src = MOUNT_SRC_VDDK70_LIB_DIR_V2V_EXAMPLE
```
Comment (xiaodwan) [inline virttest/backends/v2v/cfg/convert_source.cfg:40]: Why this diff looks strange?

File: virttest/backends/v2v/cfg/convert_source.cfg (hunk 1 above)
Comment (vwu-vera) [reply virttest/backends/v2v/cfg/convert_source.cfg:46]: Just ask mxie to copy vddk8.0.0 to NFS. Will update to 8.0 then.

**Review by xiaodwan — APPROVED** [review]

**Comment by xiaodwan** [comment]
@vwu-vera Please ask @chunfuwen to merge it after the checks, thanks.

**Comment by vwu-vera** [comment]
@chunfuwen Please help to merge this PR. Thanks.

**Review by chunfuwen — APPROVED** [review]
LGTM

**Pull request merged**